monai/bundle/config_item.py

```diff
@@ -308,7 +310,7 @@ class ConfigExpression(ConfigItem):
 
     """
 
-    prefix = "$"
+    prefix = EXPR_KEY
```

Comment:
Hi @wyli ,

If already defined the key in `utils`, do we still need this `prefix` member variable?
Same in all the other places.

Thanks.